allenact/utils/experiment_utils.py

```diff
@@ -1170,3 +1191,33 @@ def current_stage_losses(
             )
             for loss_name in self.current_stage.loss_names
         }
+
+
+def download_checkpoint_from_wandb(
```

Comment:
It LGTM. I'm not sure why we don't download directly to ckpt_dir, but I assume the wandb api function downloads more stuff than just the checkpoint?

Reply:
good point! let me actually make is download to `ckpt_dir` if it exists

Reply:
ok, I have a second look. The reason why download to `/tmp` and then move to `ckpt_dir` is because the downloaded ckpts are named `ckpt.pt`, no matter what's the step. We have to change its name anyway.
I've made it download to `ckpt_dir` directly and change the name in this folder only.